specification/advisor/resource-manager/Microsoft.Advisor/stable/2023-01-01/advisor.json

```diff
@@ -248,6 +310,74 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.Advisor/configurations/v2/{configurationName}": {
+      "put": {
+        "tags": [
+          "Configurations"
+        ],
+        "summary": "Create/Overwrite Azure Advisor configuration.",
+        "operationId": "Configurations_CreateInResourceGroupDuration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
```

Comment:
>apiVersionParameter [](http://example.com/codeflow?start=34&length=19)

Consider instead referencing "ApiVersionParamter" definition in common-types types.json Parameters section everywhere you need an api version parameter